.travis.yml

```diff
@@ -10,8 +10,8 @@ rvm:
   - 2.2
   - 2.3.0
   - ruby-head
-  - jruby-1.7
-  - jruby-9
+  - jruby-1.7.26
+  - jruby-9.1.6.0
```

Comment:
Do we need this change?  I don't like to hard-code minor versions.  For jruby-9.1.x you could just say `jruby`, if `jruby-9` pointed to jruby-9.0.x.